README.md

```diff
@@ -25,11 +25,7 @@ if err != nil {
 }
 
 
-redisHOST, err := discovery.Host("redis", "tcp")
-if err != nil {
-    logger.Fatal("ERROR: " + err.Error())
-}
-redisPORT, err := discovery.Port("redis", "tcp")
+redisHostPort, err := discovery.HostPort("redis", "tcp")
```

Comment:
Is the README meant to be an exhaustive list of all the methods? I was assuming that the API documentation was for that, and I was just changing the README to show the two that I was anticipating would be most common.